src/shared/components/case-editor/case-edit-page/case-edit-page.component.spec.ts

```diff
@@ -522,6 +522,7 @@ describe('CaseEditPageComponent', () => {
       spyOn(caseEditComponentStub, 'form');
       spyOn(caseEditComponentStub, 'validate').and.returnValue(of(validateResult));
       spyOn(formValueService, 'sanitise').and.returnValue(eventData);
+      spyOn(formValueService, 'sanitiseDynamicLists').and.returnValue(eventData);
```

Comment:
can we add a verifycation that this method is called?

Reply:
called now